### ISSUE TITLE
Cryptocurrency Bug: triggers not working

### ISSUE THREAD

**Issue by kewde** [duckduckgo/zeroclickinfo-spice#3036]
## Type of Issue
- [x] Bug (incorrect behavior)
  - [x] Triggering
  - [ ] Relevancy
  - [ ] Internal (issue with code)
- [ ] Feature Request
  - [ ] Triggers (new triggers)
  - [ ] Design Change
## Description

The cryptocurrency IA is not being triggered, ever.
The triggering had issues before when I did a little work on it, but it somehow magically went back to working.
Now, it's completely broken.
As a point of reference: https://github.com/duckduckgo/zeroclickinfo-spice/commits/master/share/spice/cryptocurrency/cryptocurrencylist.txt
It got bricked after my commits.
## Steps to recreate

1 btc to ltc
10 ltc to btc
20 sdc
10 sdc to btc

@claytonspinner 

------ 

IA Page:  https://duck.co/ia/view/cryptocurrency

**Comment by claytonspinner** [comment]
hm, not sure what would cause this to stop triggering. It works for me in the DuckDuckHack environment. I'll keep looking into it. @MrChrisW and @moollaza do you know what might cause this to work in the Codio dev environment but stop working in production?

**Comment by stormwindy** [comment]
I have tried many conversions. Only the triggers which required to convert sdc to any other currency type did not work. Others worked well.
 For example,
**These triggers work:**

1. btc to 'any currency'
2. ltc to 'any currency'
3. XRP to 'any currency'
4. 'Any currency type' to sdc

**This trigger does not work:**

1. sdc to 'any currency type'

I do not know the system much right know but I will check cheat sheet to see whether I can find something or not.

**Comment by stormwindy** [comment]
I have recreated the situation. However, everything worked well. What I have realized is that, conversion does not trigger while trying to convert 1 sdc to any other currency form. Other conversions worked well (including ltc, btc, XRP). It does not get bricked. It just gives "no results were returned from Instant Answer". 
**Example of a not working trigger**
1 sdc to 'any other currency'.
@claytonspinner
